<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogViewStatusAlarms</class>
 <widget class="QDialog" name="DialogViewStatusAlarms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Status and Alarms</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="teAlarms">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlarms">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sensor Alarms</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPidHlt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PID controller HLT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tePidHlt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>180</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="tePidBk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>180</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPidBk">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PID controller Boil-kettle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogViewStatusAlarms</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogViewStatusAlarms</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
